sdv/constraints/tabular.py

```diff
@@ -379,3 +381,73 @@ def reverse_transform(self, table_data):
         table_data[self._column] = self._formula(table_data)
 
         return table_data
+
+
+class Rounding(Constraint):
+    """Round a column based on the specified number of digits.
+
+    Args:
+        column (list[str]):
```

Comment:
column`s`

Reply:
If it is a single column, should we support being past a single string too, instead of a list?